src/main/resources/static/_shared.yml

```diff
@@ -3719,6 +3733,10 @@ components:
         - approved_premises_space_planning_dry_run
         - approved_premises_import_delius_booking_management_data
         - approved_premises_update_space_booking
+    SeedFromExcelFileType:
+      type: string
+      enum:
+        - approved_premises_room_from_excel
```

Comment:
can drop _from_excel here i think? given it's of type `SeedFromExcelFileType`

Reply:
Have removed `_from_excel`.